code/modules/spells/spell_types/teleport/blink.dm

```diff
@@ -17,3 +17,8 @@
 	outer_tele_radius = 6
 
 	post_teleport_sound = 'sound/effects/magic/blink.ogg'
+
+/datum/action/cooldown/spell/teleport/radius_turf/blink/slow
+	name = "Minor Blink"
+	desc = "This spell randomly teleports you a short distance, you're still practising doing it quickly."
```

Comment:
```suggestion
	desc = "This spell randomly teleports you a short distance, you're still practicing doing it quickly."
```

Reply:
oh wait is this a English (UK) thing?

Reply:
To be honest I also actually don't know why I wrote it with an s when I would usually use a c